src/pages/editor/PublicationChannelTableRow.tsx

```diff
@@ -0,0 +1,37 @@
+import { Chip, TableCell, TableRow } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import { useFetchOrganization } from '../../api/hooks/useFetchOrganization';
+import { useFetchPublisher } from '../../api/hooks/useFetchPublisher';
+import { ClaimedChannel } from '../../types/customerInstitution.types';
+import { getIdentifierFromId } from '../../utils/general-helpers';
+import { getLanguageString } from '../../utils/translation-helpers';
+
+interface ChannelClaimTableRowProps {
+  channel: ClaimedChannel;
+}
+export const ChannelClaimTableRow = ({ channel: claim }: ChannelClaimTableRowProps) => {
+  const { t } = useTranslation();
+  const publisher = useFetchPublisher(getIdentifierFromId(claim.channelClaim.channel));
+  const publisherName = publisher ? publisher.data?.name : '';
+
+  const organization = useFetchOrganization(claim.claimedBy.organizationId);
+  const organizationName = organization ? getLanguageString(organization.data?.labels) : '';
+
+  return (
+    <TableRow key={claim.id}>
```

Comment:
ingen array/map her, så trenger ingen key
```suggestion
    <TableRow>
```